PlagiarismSettingsForm.php

```diff
@@ -60,8 +66,8 @@ function fetch($request, $template = null, $display = false) {
 	 * @copydoc Form::execute()
 	 */
 	function execute(...$functionArgs) {
-                $this->_plugin->updateSetting($this->_contextId, 'ithenticateUser', trim($this->getData('ithenticateUser'), "\"\';"), 'string');
-		$this->_plugin->updateSetting($this->_contextId, 'ithenticatePass', trim($this->getData('ithenticatePass'), "\"\';"), 'string');
+		$this->_plugin->updateSetting($this->_contextId, 'ithenticateUser', trim($this->getData('ithenticateUser'), "\"\';"), 'string');
+		$this->_plugin->updateSetting($this->_contextId, 'ithenticatePass', trim($this->getData('ithenticatepass'), "\"\';"), 'string');
```

Comment:
It should be be `ithenticatePass` instead of `ithenticatepass` that is with capital `P` .